src/js/reducers/types.js

```diff
@@ -0,0 +1,11 @@
+export const PLAY_PAUSE_SELECTED_TAKE='PLAY_PAUSE_SELECTED_TAKE';
+export const PAUSE_SELECTED_TAKE='PAUSE_SELECTED_TAKE';
+export const LOADING='LOADING';
+export const FETCH_ALTERNATE_TAKES='FETCH_ALTERNATE_TAKES';
+export const FETCH_CHUNKS='FETCH_CHUNKS';
+export const UPDATE_ACTIVE_CHUNK_INDEX = 'UPDATE_ACTIVE_CHUNK_INDEX';
+export const SWAP_TAKE = 'SWAP_TAKE';
+export const UNDO_SWAP = 'UNDO_SWAP';
+export const UPDATE_ALTERNATE_TAKES = 'UPDATE_ALTERNATE_TAKES';
+export const FINISH_PLAYING = 'FINISH_PLAYING';
+export const CLEAR_ALTERNATE_TAKES = 'CLEAR_ALTERNATE_TAKES';
```

Comment:
Question: the idea of having a types const is to import them on reducers and actions, to avoid typos when we are returning the Type of action and the  switch case in the reducer, are you following that pattern? if so, there is already a file named types.js in the actions folder, we tried to follow that pattern but at some point we stopped it. I think it is a good practice to follow, but to keep it consistent let's have only one types.  so you can add the other const to this file and make the refactor or add these const to the types.js file in the actions folder.  what do you think about it?

Reply:
Yes I do think that it is a good pattern to follow and I agree with you that it is best to have them in one location. I didn't put it with the actions because it would have been weird to be calling these Reducer Constants from the actions folder. I think if we combined them and moved them into a folder under the js/ directory and named it something like reduxConstants.js or something that would be a good idea